### PR TITLE
Use golang for elb

### PR DESCRIPTION
**Pull request by zengchen1024**
This pr depends on #38 #40 

--- PASS: TestAccELBListener_basic (93.27s)
PASS
ok      github.com/terraform-providers/terraform-provider-opentelekomcloud/opentelekomcloud     93.279s

--- PASS: TestAccELBBackend_basic (160.00s)
PASS
ok      github.com/terraform-providers/terraform-provider-opentelekomcloud/opentelekomcloud     160.007s

--- PASS: TestAccELBHealth_basic (126.45s)
PASS
ok      github.com/terraform-providers/terraform-provider-opentelekomcloud/opentelekomcloud     126.457s

--- PASS: TestAccELBLoadBalancer_basic (120.18s)
PASS
ok      github.com/terraform-providers/terraform-provider-opentelekomcloud/opentelekomcloud     120.188s

### CODE REVIEW

**Comment by zengchen1024** [comment]
@niuzhenguo Could you take at this pr? Thanks!

File: opentelekomcloud/config.go [hunk 1]
```diff
@@ -377,10 +377,10 @@ func (c *Config) objectStorageV1Client(region string) (*gophercloud.ServiceClien
 	})
 }
 
-func (c *Config) otcV1Client(region string) (*gophercloud.ServiceClient, error) {
-	return openstack.NewOtcV1(c.OsClient, gophercloud.EndpointOpts{
+func (c *Config) otcV1Client(region string) (*golangsdk.ServiceClient, error) {
+	return huaweisdk.NewElbV1(c.HwClient, golangsdk.EndpointOpts{
```
Comment (niuzhenguo) [inline opentelekomcloud/config.go]: I find we also have a NewElasticLoadBalancer in golangsdk, which seems following more conventions and no need to pass a "elb" parameter. how about changing to use that?

Comment (zengchen1024) [reply opentelekomcloud/config.go]: I see. I will update. Thanks!

Comment (zengchen1024) [reply opentelekomcloud/config.go]: Done

**Pull request merged**